messages/EventKeyTemplates.js

```diff
@@ -7,7 +7,15 @@ module.exports = {
 		"customEslintHeadsUp": "About to run Eslint with custom config in %s. Please make sure your current directory has all the required NPM dependencies.",
 		"customPmdHeadsUp": "About to run PMD with custom config in %s. Please make sure that any custom rule references have already been added to the plugin through scanner:rule:add command.",
 		"pmdRuleSkipped": "Omitting results for PMD rule \"%s\". Reason: %s.",
-		"unmatchedPathExtensionCpd": "Path extensions for the following files will not be processed by CPD: %s"
+		"unmatchedPathExtensionCpd": "Path extensions for the following files will not be processed by CPD: %s",
+		"sfgeInfoLog": "%s",
+		"sfgeMetaInfoCollected": "Loaded %s: [ %s ]",
+		"sfgeFinishedCompilingFiles": "Compiled %s files.",
+		"sfgeStartedBuildingGraph": "Building graph . . .",
+		"sfgeFinishedBuildingGraph": "Added all compilation units to graph.",
+		"sfgePathEntryPointsIdentified": "Identified %s path entry point(s).",
+		"sfgeViolationsInPathProgress": "Detected %s violation(s) from %s path(s) on %s entry point(s) so far.",
+		"sfgeCompletedPathAnalysis": "Overall, analyzed %s path(s) from %s entry point(s). Detected %s violation(s)."
```

Comment:
New messages to be displayed during verbose calls.
@teresa-allen-sfdc Please do let me your thoughts on the verbose messages.